draft-ietf-quic-qpack.md

```diff
@@ -162,12 +177,45 @@ the dynamic table is less than or equal to the maximum size.
 This mechanism can be used to completely clear entries from the dynamic table by
 setting a maximum size of 0, which can subsequently be restored.
 
-### Absolute and Relative Indexing {#indexing}
+### Calculating Table Size
+
+The size of the dynamic table is the sum of the size of its entries.
+
+The size of an entry is the sum of its name's length in octets (as defined
+in {{string-literals}}), its value's length in octets, and 32.
+
+The size of an entry is calculated using the length of its name and value
+without any Huffman encoding applied.
+
+### Absolute Indexing
 
 Each entry possesses both an absolute index which is fixed for the lifetime of
 that entry and a relative index which changes over time based on the context of
-the reference. The first entry inserted has an absolute index of "1"; indices
-increase sequentially with each insertion.
+the reference.
+
+The absolute index takes on the values in the range
+
+~~~
+   [ 0, MaxEntries * 2 - 1 ]
```

Comment:
Inclusive, I presume.

Reply:
Yes, inclusive.  Is something more that using square brackets required?